### ISSUE TITLE
Barron Geddon in Molten Core does NOT drops the legendary item,Even after 100 kills...!!!!!!

### ISSUE THREAD

**Issue by Cromanthus** [dalaranwow/dalaran-wow#5194]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I have been farming Baron Geddon for his Legendrary drop for 5 months now with over 100 kills and it has still not dropped it yet

**Expected behaviour**: The drop rate is around 3 % so after over 100 kills it should have dropped at least 3 times...but not even once !!

**Steps to reproduce the problem**:

1. Kill Barron Geddon over 100 times
2. The legendary item does NOT drop.
3. 

**Include proofs for this behaviour**

You can research this on several pages like https://wotlk.evowow.com/?item=18563
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
#Molten core

**Comment by Bajeru** [comment]
Isnt molten core weekly reset, in that case im pretty sure 5 months isnt 100 weeks = 100 runs else i failed my math in school badly

**Comment by Crispybuilt** [comment]
Drops fine.
Closing this.

**Comment by Cromanthus** [comment]
When you killed baron you did it so you got saved OR you timed it so Baron always kills him self ?
